.github/workflows/pipeline.yml

```
@@ -0,0 +1,327 @@
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-region: ${{ env.AWS_REGION }}
+
+    - name: Rollback to previous version
+      uses: ./.github/actions/rollback
+      with:
+        ec2-host: ${{ secrets.EC2_HOST }}
+        ec2-username: ${{ secrets.EC2_USERNAME }}
+        ec2-ssh-key: ${{ secrets.EC2_SSH_KEY }}
+        ecr-registry: ${{ needs.build.outputs.registry }}
```

actionlint (1.7.7)

property "build" is not defined in object type {deploy-production: {outputs: {}; result: string}}

(expression)